extensions/wikia/AdEngine/js/provider/gpt/helper.js

```diff
@@ -94,6 +89,31 @@ define('ext.wikia.adEngine.provider.gpt.helper', [
 			googleApi.addSlot(element);
 		}
 
+		function setAdditionalTargeting(slotTargetingData) {
```

Comment:
Actually, it is just a first step -- in ideal world we need to refactor this module and move this logic to separate module.